aikau/src/test/resources/alfresco/lists/AlfSortablePaginatedListTest.js

```diff
@@ -23,10 +23,51 @@
 define(["intern!object",
         "intern/chai!assert",
         "require",
-        "alfresco/TestCommon"], 
-        function (registerSuite, assert, require, TestCommon) {
+        "alfresco/TestCommon",
+        "intern/dojo/node!leadfoot/keys"], 
+        function (registerSuite, assert, require, TestCommon, keys) {
+
+   
 
    var browser;
+
+   var testClearingDocumentList = function(buttonId, errorMsg) {
+      return browser.findByCssSelector(".alfresco_logging_DebugLog__clear-button")
+         .click()
+      .end()
+      .findByCssSelector("#DOCUMENT_LIST tr:nth-child(1) .alfresco-renderers-Property")
+         .click()
+         .pressKeys(keys.ARROW_DOWN)
+         .pressKeys(keys.ARROW_DOWN)
+         .pressKeys(keys.ARROW_DOWN)
+         .pressKeys(keys.ARROW_DOWN)
+         .pressKeys(keys.ARROW_DOWN)
+         .pressKeys(keys.ARROW_DOWN)
+         .pressKeys(keys.ARROW_DOWN)
+         .pressKeys(keys.ARROW_DOWN)
+      .end()
+      .getLastPublish("DOCUMENT_LIST_ALF_EVENTS_SCROLL")
```

Comment:
Yes, it should ensure that you don't count the the number of results displayed until after they've been loaded and it also checks that the events scroll is being generated - I'm assuming (although I haven't checked) that if the publication doesn't exist then some kind of error will occur? Or do I have to explicitly check the returned object?